client/my-sites/plugins-wpcom/business-plugins-panel.jsx

```diff
@@ -18,17 +19,24 @@ const defaultPlugins = [
 
 export const BusinessPluginsPanel = React.createClass( {
 	render() {
-		const { plugins: givenPlugins = [] } = this.props;
+		const {
+			isActive = false,
+			plugins: givenPlugins = []
+		} = this.props;
 		const plugins = givenPlugins.length
```

Comment:
We have the default above in destructuring. Can we make use of that and get rid of this ternary?

```
const {
	isActive = false,
 	plugins: givenPlugins = defaultPlugins
} = this.props
```
Is there a possibility that someone passes an empty array as a prop?
Should it be overwritten by defaultPlugins then?